// @ali-sanad/random-fruits/assets/fruits-data.js
exports.fruits = {
  A: ['Apples', 'Apricots', 'Avocados'],
  B: ['Bananas', 'Boysenberries', 'Blackberries', 'Blueberries', 'Bing Cherry'],
  C: ['Cherries', 'Cantaloupe', 'Crab apples', 'Clementine', 'Cucumbers'],
  D: [
    'Damson plum',
    'Dinosaur Eggs, Dates',
    'Dewberries',
    'Dragon Fruit, Durian',
  ],
  E: ['Elderberry', 'Eggfruit', 'Evergreen Huckleberry', 'Entawak'],
  F: ['Fig', 'Farkleberry', 'Finger Lime'],
  G: ['Grapefruit', 'Grapes', 'Gooseberries', 'Guava'],
  H: ['Honeydew melon', 'Hackberry', 'Honeycrisp Apples'],
  I: ['Indian Prune', 'Indonesian Lime', 'Imbe', 'Indian Fig'],
  J: ['Jackfruit', 'Java Apple', 'Jambolan'],
  K: ['Kiwi', 'Kaffir Lime', 'Kumquat'],
  L: ['Lime', 'Lemon', 'Longan', 'Lychee', 'Loquat'],
  M: ['Mango', 'Mandarin Orange', 'Mulberry, Melon'],
  N: ['Nectarine', 'Navel Orange', 'Nashi Pear (Asian Pear)'],
  O: ['Olive', 'Oranges', 'Ogeechee Limes', 'Oval Kumquat'],
  P: [
    'Papaya',
    'Persimmon',
    'Plum',
    ' Paw Paw',
    'Prickly Pear',
    'Peach',
    'Pomegranate',
    'Pineapple',
    'Passion Fruit',
  ],
  Q: ['Quince', 'Queen Anne Cherry', 'Quararibea cordata (Chupa Chupa)'],
  R: ['Rambutan', 'Raspberries', 'Rose Hips'],
  S: ['Star Fruit', 'Strawberries', ' Sugar Baby Watermelon'],
  T: ['Tomato', 'Tangerine', 'Tamarind', ' Tart Cherries'],
  U: ['Ugli Fruit', 'Uniq Fruit', 'Ugni'],
  V: ['Vanilla Bean', 'Velvet Pink Banana', 'Voavanga'],
  W: ['Watermelon', 'Wolfberry', 'White Mulberry'],
  X: [
    'Xigua (Chinese Watermelon)',
    'Ximenia caffra fruit',
    'Xango Mangosteen Fruit Juice',
  ],
  Y: ['Yellow Passion Fruit', 'Yunnan Hackberry', 'Yangmei'],
  Z: ['Zig Zag Vine fruit', 'Zinfandel Grapes', 'Zucchini'],
};
